cmd/storage-server/run.go

```diff
@@ -76,22 +78,37 @@ func Start(logger *log.Logger, port int, storePath string) (wait func(), shutdow
 		return nil, nil, err
 	}
 
-	store := diskv.New(diskv.Options{
-		BasePath:     storePath,
-		CacheSizeMax: 1024 * 1024, // 1MB cache size
-		AdvancedTransform: func(s string) *diskv.PathKey {
-			path := strings.Split(s, "/")
-			last := len(path) - 1
-			return &diskv.PathKey{
-				Path:     path[:last],
-				FileName: path[last],
-			}
-		},
-		InverseTransform: func(pk *diskv.PathKey) string {
-			return strings.Join(pk.Path, "/") + pk.FileName
-		},
-	})
-	storageServer := newStorageServer(logger, store)
+	var storageSinks []Store
+
+	gdriveStore := &gdriveServer{}
+	err = gdriveStore.Setup(logger)
+	if err != nil {
+		logger.Errorf("Unable to setup Google Drive: %v", err)
+	} else {
+		storageSinks = append(storageSinks, gdriveStore)
+	}
+
+	if !storeRemotely {
```

Comment:
We can't store remotely and store locally?

Reply:
We can, it's default behaviour. The flag is meant for the case where local storage space is low so store everything remotely.